src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/patternvariablename/InputPatternVariableNameEnhancedInstanceof.java

```diff
@@ -0,0 +1,88 @@
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.naming.patternvariablename;
+
+import java.util.ArrayList;
+import java.util.Locale;
+
+public class InputPatternVariableNameEnhancedInstanceof {
+    private Object obj;
+
+    static boolean doStuff(Object obj) {
+        // Violation, OTHER doesn't match ^[a-z][a-zA-Z0-9]*$
+        return obj instanceof Integer OTHER && OTHER > 0;
+    }
+
+    static {
+        Object o = "";
+        //Violation, s doesn't match ^[a-z][a-zA-Z0-9]+$
```

Comment:
```suggestion
        // Violation, s doesn't match ^[a-z][a-zA-Z0-9]+$
```

Meta comment: do we typically list "violation" and "ok" comments in test inputs that are used in multiple test configurations? I was just thinking it's potentially a little confusing reading this file to see references to different patterns.

Reply:
Input should be used in single test methods, so single config.
Unfortunately not all inputs follow this model.

Reply:
Fixed.